### ISSUE TITLE
`AudiobookBase.narrators` is wrong

### ISSUE THREAD

**Issue by miniBill** [sonallux/spotify-web-api#229]
It's

```yaml
type: array
$ref: '#/components/schemas/NarratorObject'
```

instead of

```yaml
type: array
items:
  $ref: '#/components/schemas/NarratorObject'
```

**Comment by sonallux** [comment]
Thank you for creating this issue. This is fixed in version `2023.2.26`, which I have just released.

**Comment by miniBill** [comment]
Thank you for your work 💜
